### ISSUE TITLE
Applying image to prototype token broken in v12

### ISSUE THREAD

**Issue by jacobsteffler** [Aedif/TokenVariants#181]
**Describe the bug**
I enable the "Transfer Token Updates to Prototype" option. This worked up until recently, and now no longer works for any token. When switching token images using the HUD, the linked actor's prototype token's image is not updated.

This **is** working as expected in a fresh world. So it's not necessarily a simple bug in the module.
I'd appreciate any help investigating what may be going on in my world.
I have also encountered the issue with fresh actors within my world, so it's something with my world and not the actors.

No messages appear in the console when performing the workflow.

**To Reproduce**
1. Enable TokenVariants with the libWrapper dependency
2. In the settings, enable Transfer Token Updates to Prototype
3. Create a new Player Character actor where the prototype is linked, and select a default image
4. Use the TokenVariants HUD to switch to a different token image
5. See that the prototype token image was not changed

**Desktop (please complete the following information):**
- OS/Browser independent
- Foundry 12.331
- DND5e 3.3.1
- Token Variant 5.1.5 (also tested 5.1.3)
- libWrapper 1.13.2.0
- All other modules disabled

**Comment by jacobsteffler** [comment]
I investigated this, and have more details to share.

First, I tried again in a fresh world, and this time I was able to reproduce the issue consistently. I'm not sure if I was just mistaken the last time.

Anyway, I traced this down to Foundry's code document update code, where it will clean any objects it is passed:
![Screenshot_20241206_220830](https://github.com/user-attachments/assets/30353460-b4c1-4897-b373-40bbea515f90)

Part of this cleaning involves stripping unknown keys from objects. The unknown key deleted here was "token":
![Screenshot_20241206_222744](https://github.com/user-attachments/assets/527b21dc-3014-4e32-b6ba-e1f55b521e6e)

The actual expected data model key for the prototype token looks to be "prototypeToken":
![Screenshot_20241206_223102](https://github.com/user-attachments/assets/d0723c56-c9a9-4b66-aa3b-9184db9d2865)

I'm not sure exactly what/when changed here. We've been on Foundry v12 for a while now, so I don't think this was tied to a core API change.
